.github/workflows/main.yml

```diff
@@ -0,0 +1,86 @@
+name: Build Linux
+
+on:
+  push:
+    branches:
+      - jellyfin
+
+  pull_request:
+    branches:
+      - jellyfin
+
+  release: 
+    types:
+      - released
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build ${{ matrix.release }} ${{ matrix.arch }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        release:
+          - bullseye
+          - buster
+          - stretch
+          - groovy
+          - focal
+          - bionic
+
+        arch:
+          - amd64
+          - arm64
+          - armhf
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install make and mmv
+        run: sudo apt-get install make mmv
+
+      - name: Install Docker
+        uses: docker-practice/actions-setup-docker@v1
```

Comment:
interesting that this is needed as the virtual environment for the actions has docker, as well as podman, installed.

https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#tools

Note: this is not blocking

Reply:
Good catch! It was a long time since I looked at this. I'll see if that can be removed. 

The failing build was for Debian stretch. I wanted to migrate it exactly like it was so I haven't removed it (yet).

Reply:
sounds like a plan, then adding bullseye should happen too in that followup :+1: